bloom/filter_test.go

```diff
@@ -95,6 +95,68 @@ func TestFilterInsert(t *testing.T) {
 	}
 }
 
+// TestFilterFPRange checks that new filters made with out of range
+// false positive targets result in either max or min false positive rates.
+func TestFilterFPRange(t *testing.T) {
+
+	// fprates greater than 1 should be clipped at 1, resulting in a high
+	// false positive rate and small filter size.
+	f := bloom.NewFilter(1, 0, 20.9999999769, wire.BloomUpdateAll)
+
+	inputStr := "02981fa052f0481dbc5868f4fc2166035a10f27a03cfd2de67326471df5bc041"
+	sha, err := wire.NewShaHashFromStr(inputStr)
+	if err != nil {
+		t.Errorf("TestMerkleBlockP2PubKeyOnly NewShaHashFromStr failed: %v", err)
```

Comment:
The function name in the error is incorrect, but it isn't really needed in top level test functions like this since the `go test` testing framework will print the function for you already.

Same goes for the others.